documentation/book/assembly-jvm-options.adoc

```diff
@@ -13,7 +14,7 @@
 
 = JVM Options
 
-Apache Kafka and Apache Zookeeper run inside a Java Virtual Machine (JVM).
+Apache Kafka, Apache Zookeeper, Apache Kafka Connect, Apache Kafka Mirror Maker and the Kafka Bridge run inside a Java Virtual Machine (JVM).
```

Comment:
The extra components make this sentence quite difficult to parse. Please use a bulletted list instead:

The following components of {ProductName} run inside a Virtual Machine (VM):

* Apache Kafka
* Apache Zookeeper
* Apache Kafka Connect
* Apache Kafka Mirror Maker
* {ProductName} Kafka Bridge